core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,39 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
+@media(max-width:768px) {
+/* Disabling navbar-collapse */
+  .oppia-navbar-breadcrumb>li, .oppia-navbar-breadcrumb {
+    float: left;
+  }
+  .oppia-navbar-breadcrumb {
+    margin-top: 0;
+    margin-left: 5px;
+  }
+  .navbar-nav.oppia-navbar-nav > li > a{
+    display: none;
+  }
+}
+@media(max-width:600px) {
+  .oppia-navbar-breadcrumb-icon {
```

Comment:
Thanks @seanlip, no it didn't make any sense as navbar-collapse was intact in other pages. So this was not needed globally. Will move it to exploration player file.